src/devices/namron.ts

```diff
@@ -942,6 +942,25 @@ const definitions: Definition[] = [
             await reporting.activePower(endpoint);
         },
     },
+    {
+        zigbeeModel: ['4512772', '4512773'],
+        model: '4512773',
+        vendor: 'Namron',
+        description: 'Zigbee Kanalbryter Sort',
```

Comment:
Please make the descriptions English

Reply:
Sorry about that, i used the product name in this case.
should be fixed now